app/models/loan.rb

```diff
@@ -81,9 +82,23 @@ def self.filter_by_params(params)
     params.reverse_merge! self.default_filter
     scoped = self.all
     scoped = scoped.status(params[:status]) if params[:status] != 'all'
+
+    if self.check_before_division_filter(params[:division])
+      scoped = scoped.by_division(self.find_division_id_with(params[:division]))
+    end
+
     scoped
   end
 
+  def self.find_division_id_with(name)
+    Division.find_by(name: name).id if name
```

Comment:
Does this work? `get_division_from_url` returns a symbol like `:us` or `:argentina`, but division names are something like `La Base USA`, I wouldn't expect this to behave correctly as written when there isn't a division param passed in.

Reply:
I had an issue with git
perhaps that change wasn't added

I'll work on it
thanks for the find

Reply:
There's a check for this below but because of your comment, I've realized that the symbols are not country codes and I have updated the check accordingly

Thanks!

Reply:
The problem is that we want each division to default to showing only its own loans, so we actually _want_ it to filter by the value of `get_division_from_url` if it can -- if I'm following this correctly, right now it will not do that, and instead will default to showing all unless it specifically detects the name param from the dropdown.

That's why I kept mentioning that other `short_name` issue in conjunction with this -- if we set the short name for each division to match the param we get from `get_division_from_url` we can use the correct filtering based on that.